lib/mongoid/slug.rb

```diff
@@ -24,6 +24,13 @@ module Slug
       # alias_attribute :slugs, :_slugs
     end
 
+    class << self
+      attr_accessor :slug
+      def config(&block)
```

Comment:
Rename `config` to `configure`, that's the general convention I've seen across Ruby gems.